environment/README.md

```diff
@@ -16,6 +18,19 @@ _Lists the geological and physical environment details of collection points._
 
 * `visibility.csv` - Sky View Descriptions.
 
+#### _CALIBRATIONS_ ####
+
+| Field | Description |
+| --- | --- |
+| _Station_ | Station Code Name  
+| _Location_ | Sensor Location (e.g. Borehole or surface sensor) 
```

Comment:
MLNZ20 doesn't use location codes, it only has the correction at the station level. It's possible this is an oversight and we may be able to test this later with borehole installations which would make the loc column necessary. When SeisComP pulls data from this table, can it ignore loc column for MLNZ20 reflecting how its corrections are defined now?